cell.py

```diff
@@ -0,0 +1,16 @@
+"""doc"""
```

Comment:
why do you have these? """doc"" looks terrible and this is an unnecessary one anyways. This file clearly doesn't need describing